2024-06-24/minutes.md

```diff
@@ -1,3 +1,188 @@
+
+[Ruth] Thanks, Florian. That sounds really good. We're excited.Okay, so for example, then I think you're right, I think we need to create a spreadsheet. So we will think about that ASAP and what we will think about that ASAP and what we can do is we wanted to review that ASAP and what we can do is we wanted to review what external examples we wanted anyway because they're not all going to be able to review what external examples we wanted anyway because they're not all going to review what external examples we wanted anyway because they're not all going to be able to be live examples. So we need to figure out what's what and maybe once we've done that we will then have a list of the ones that can be moved and we can tell translated content.
+
+[Pranshu] Should we create an issue or discussion for this so that it's easier to for everybody else to see?
+
+[Ruth] Let us do the audit 1st and then we'll create a discussion which can link to the spreadsheet with the things that we want people to help us with. Yeah, cause you're right, yes we do, but I think we probably need a list first, don't we? And that we contributors can come in and take things off our list.
+
+[Pranshu] Awesome! Looking forward to some progress in the next call.
+
+[Ruth] Yeah.
+
+### Is there an objective around closing issues? (@rumyra)
+
+[Ruth] I had a question around closing issues because we have contributors closing lots of issues. I found out the answer earlier, which is there is no objective. People just wanted to come in and close loads of issues. So I guess the things. Mention here is you're gonna see a lot of activity over the next few weeks because some of our contributors are trying to reduce that issue count, which is fine. This is totally fine. Just because they want to. There was no other reason. They just wanted to do it. Which is cool.
+
+### Inline icons [discussion](https://github.com/orgs/mdn/discussions/654#discussioncomment-9746229) (@rumyra)
+[Ruth] The inline icons discussion on code. This is probably something most relevant to you, Onkar. This is the idea of putting experimental and deprecated icons throughout the site so not just in the side bars but making sure they're clear on pages and in our reference pages and things like that.
+
+I want to see the vote on the discussion to see how maintainers feel we this should be done. But I don't think at this time it's a good idea for us to implement any way to do it because of the platform changes that are coming. So we have go through and have like, including macros and doing all of that kind of stuff and in like 3 months that'll all be mute and null because we'll be doing it in a completely different way.
```

Comment:
```suggestion
I want to see the vote on the discussion to see how maintainers feel this should be done. But I don't think at this time it's a good idea for us to implement it because of the platform changes that are coming. So we will be updating macros, but in about three months, that'll all be moot because we'll be doing it in a completely different way.
```